src/type/schema.js

```diff
@@ -269,3 +278,10 @@ function typeMapReducer(map: TypeMap, type: ?GraphQLType): TypeMap {
 
   return reducedMap;
 }
+
+function getDirectiveArgTypes(directive: GraphQLDirective) {
+  if (!isDirective(directive)) {
+    return [];
+  }
+  return directive.args.map(arg => getNamedType(arg.type));
```

Comment:
@Yogu You don't need `getNamedType` here since it's already handled here:
https://github.com/Yogu/graphql-js/blob/7321a186854af3a095fe7ee52a2a1622dbf288b8/src/type/schema.js#L235-L237